build.ps1

```diff
@@ -61,7 +63,19 @@ If (CompareVersions -actualVersion $vboxVersion -expectedVersion $virtualBoxMinV
 
 }
 
-$packerVersion = cmd.exe /c $packer -v

```

Comment:
This reduces the search space significantly.  The `packer` single binary distribution model makes much more likely simply be places `somewhere` on the path vs in an expected location.

Reply:
roger